### PR TITLE
Remove strange file

### PR DESCRIPTION
**Pull request by alexey-milovidov**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

#### CI Settings (Only check the boxes if you know what you are doing):
- [ ]  Allow: All Required Checks
- [ ]  Allow: Stateless tests
- [ ]  Allow: Stateful tests
- [ ]  Allow: Integration Tests
- [ ]  Allow: Performance tests
- [ ]  Allow: All Builds
- [ ]  Allow: batch 1, 2 for multi-batch jobs
- [ ]  Allow: batch 3, 4, 5, 6 for multi-batch jobs
---
- [ ]  Exclude: Style check
- [ ]  Exclude: Fast test
- [ ]  Exclude: All with ASAN
- [ ]  Exclude: All with TSAN, MSAN, UBSAN, Coverage
- [ ]  Exclude: All with aarch64, release, debug
---
- [ ]  Run only fuzzers related jobs (libFuzzer fuzzers, AST fuzzers, etc.)
- [ ]  Exclude: AST fuzzers
---
- [ ]  Do not test
- [ ]  Woolen Wolfdog
- [ ]  Upload binaries for special builds
- [ ]  Disable merge-commit
- [ ]  Disable CI cache

### CODE REVIEW

**Comment by robot-ch-test-poll2** [comment]
*This is an automated comment for commit 80bd0cd58d31e9c6668778c6756984de3c79c97c with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/70662/80bd0cd58d31e9c6668778c6756984de3c79c97c/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Style check</td><td>Runs a set of checks to keep the code style clean. If some of tests failed, see the related log from the report</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/70662/80bd0cd58d31e9c6668778c6756984de3c79c97c/style_check.html">✅ success</a></td></tr>
<tbody>
</table>
</details>

File: docs/zh/introduction/distinctive-features.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ ClickHouse会使用服务器上一切可用的资源，从而以最自然的方
 
 ## 支持SQL {#zhi-chi-sql}
 
-ClickHouse支持一种[基于SQL的声明式查询语言](../sql-reference/index.md)，它在许多情况下与[ANSI SQL标准](../sql-reference/ansi.md)相同。
+ClickHouse支持一种基于SQL的声明式查询语言。
```
Comment (antaljanosbenjamin) [inline docs/zh/introduction/distinctive-features.md:40]: I have zero idea about Chinese and Russian. I trust you with the Russian version, will ask somebody to check this in any case.

**Review by antaljanosbenjamin — APPROVED** [review]

**Comment by alexey-milovidov** [comment]
Needs https://github.com/ClickHouse/clickhouse-docs/pull/2704

File: docs/zh/introduction/distinctive-features.md (hunk 1 above)
Comment (alexey-milovidov) [reply docs/zh/introduction/distinctive-features.md:40]: No need to bother, it looks ok in Google Translate.

**Pull request merged**